packages/fx-core/src/common/localSettingsConstants.ts

```diff
@@ -48,3 +48,10 @@ export const LocalSettingsBotKeys = Object.freeze({
   BotDomain: "botDomain",
   BotEndpoint: "botEndpoint",
 });
+
+export const LocalSettingsEncryptKeys = Object.freeze({
+  Crypto: "crypto",
```

Comment:
`Crypto` is not the config key name, just a constant used as secret prefix, so moving it out from `LocalSettingsEncryptKeys` would be better.

Reply:
Fixed.

Reply:
Fixed.